src/components/navbar/Navbar.jsx

```diff
@@ -0,0 +1,120 @@
+//navbar
+import { Fragment } from "react";
+import { Disclosure, Menu, Transition } from "@headlessui/react";
+import { Bars3Icon, BellIcon, XMarkIcon } from "@heroicons/react/24/outline";
+
+const navigation = [
+  { name: "Dashboard", href: "#", current: true },
+  { name: "Team", href: "#", current: false },
+  { name: "Projects", href: "#", current: false },
+  { name: "Calendar", href: "#", current: false },
+];
+
+function classNames(...classes) {
```

Comment:
Please ensure consistency in your function declarations. If you're using arrow functions, it's better to stick with that style throughout your code. This consistency helps in maintaining readability and uniformity in the codebase